dashboard/top-sql.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Top SQL
+summary: This document describes how to use Top SQL to locate SQL queries that contribute to a high load.
+---
+
+# Top SQL
+
+> **Warning:**
+>
+> Currently, Top SQL is an experimental feature. It is not recommended that you use it for production environments.
+
+This document describes how to use Top SQL to locate SQL queries that contribute to a high load of a TiDB or TiKV node in a specified time range. For example, you can use Top SQL to locate an analytic query that consumes 99% of the load for a low-load database.
+
+For a specified TiDB or TiKV node, Top SQL provides the following features:
+
+* Show the top 5 types of SQL queries that contribute the most to the load in a specified time range.
+* Show information such as CPU usage, requests per second, average latency, and query plan of a particular query, which can be used for potential performance optimization to improve your business.
+
+## Enable Top SQL
+
+The Top SQL feature is disabled by default. You can enable the feature for the entire cluster using either of the following methods:
+
+- Method 1: Log in to TiDB Dashboard, click **Top SQL** in the left pane, click the gear button in the upper-right corner of the page, and then enable the Top SQL feature.
+- Method 2: Set the value of the TiDB system variable [`tidb_enable_top_sql`](/system-variables.md#tidb_enable_top_sql-introduced-from-v540-version) to `ON`.
+
+> **Note:**
+>
+> Enabling Top SQL has a slight impact on the performance of your cluster.
+
+## Use Top SQL
+
+Once Top SQL is enabled, you can log into TiDB Dashboard, and then click **Top SQL** in the left pane to use it.
+
+![Top SQL](/media/dashboard/top-sql-overview.png)
+
+Usage tips：
+
+* You can select the target node and time range in the drop-down lists at the top of the page, or you can select the time range in the chart.
```

Comment:
```suggestion
* You can select the target node and time range in the drop-down menu at the top of the page, or you can select the time range in the chart.
```

We usually call it "drop-down menu"

Reply:
Do you have a refererence? I searched "drop-down menu" in our TiDB English docs and got no results.

Reply:
https://getbootstrap.com/docs/4.0/components/dropdowns/

Reply:
Thanks. I see "dropdown-menu" is used at the code level for R&D but we are using "drop-down list" in user documents usually for users. See the IBM Style Guide as follows:

![image](https://user-images.githubusercontent.com/79440533/150900927-3f0c2f71-e243-4da3-9543-a50e480a88f5.png)